games-emulation/ryujinx/ryujinx-1.1.1093.ebuild

```diff
@@ -328,21 +329,21 @@ RDEPEND="
 	media-libs/libsdl2
 	x11-libs/gtk+:3
 "
-BDEPEND="
-	>=dev-dotnet/dotnet-sdk-bin-7.0.401:${DOTNET_PKG_COMPAT}
-"
 
 CHECKREQS_DISK_BUILD="3G"
 DOTNET_PKG_PROJECTS=( "src/${PN^}/${PN^}.csproj" )
 
+PATCHES=(
+	"${FILESDIR}/${P}-better-defaults.patch"
+	"${FILESDIR}/${P}-disable-updates.patch"
+	"${FILESDIR}/${P}-logs-path.patch"
```

Comment:
In all of those could you also use "${PN}-1.1.1093"? This will make it easier to do bumps if we can reuse the patches.